### ISSUE TITLE
mehr Felder Teil 2

### ISSUE THREAD

**Issue by XHalbert** [cmsimple-xh/xhshop#191]
Bei dem geschlossenen #15 waren noch ein paar zusätzliche Felder in den Kontaktangaben genannt.
Sind die untergegangen? Ich finde sie nicht mehr... Auch Christoph fand:

> Ich denke ein paar Felder sollten ergänzt werden (abweichende Lieferadresse erscheint mir z.B. sinnvoll),

(ach ja, da war doch was mit den Auslandsversandkosten ...?...)
Ich für meinen Teil vermisse die abweichende Lieferadresse schon, und mind. eine Checkbox ("eilige Bestellung") wär nicht schlecht, auch als Vorlage, um weitere Checkboxen bei Bedarf einzubauen..

Wie seht ihr das?

**Comment by cmb69** [comment]
> Bei dem geschlossenen #15 waren noch ein paar zusätzliche Felder in den Kontaktangaben genannt.

Stimmt. Danke fürs Bemerken. :)

> Wie seht ihr das?

Es wird vermutlich sehr schwer es allen recht zu machen. Felder, die es bereits gibt, zu entfernen ist zwar leichter als neu hinzuzufügen, aber insgesamt sollten wir es nicht übertreiben. Vielleicht sollte dieses Thema im Forum angesprochen werden, wo doch vermutlich mehr Nutzer Feedback geben würden.

**Comment by XHalbert** [comment]
diese Entscheidung betr. Lieferadresse verstehe ich nicht, die dürfte wohl jeder brauchen... mir egal, bau ich halt selber ein

**Comment by Hartmut63** [comment]
Ich würde zusätzliche Felder für die zusätzliche Lieferschrift auch befürworten.
Bei meinem (Wellrad) Onlineshop wurde bei ca. 5 - 10 % der Bestellungen eine abweichende Lieferanschrift angegeben. Gründe waren dafür beispielsweise:

- Ein onlineversierter Sohn / Tochter bestellt für die betagten Offline Eltern / Großeltern"
- Die Lieferung erfolgt an den Arbeitsplatz, da Arbeitszeit und Lieferzeiten / Postfilialen Öffnungszeiten nicht vereinbar sind
- Das Bestellung soll als Geschenk direkt zugestellt werden.

Eine kleine Anmerkung an das Kontaktformular:
Das Wellrad Kontaktformular war wesentlich kompakter gewesen da bspw. PLZ und Ort in einer Zeile waren.
Lässt sich dies bei XH-Shop auch realisieren?

Hartmut

**Comment by cmb69** [comment]
> diese Entscheidung betr. Lieferadresse verstehe ich nicht

Welche Entscheidung? Ich habe doch lediglich vorgeschlagen, das Thema zusätzliche Felder vielleicht besser auch/zunächst im Forum zu besprechen. Grundsätzlich habe ich ja nichts gegen eine abweichende Lieferanschrift.

**Comment by XHalbert** [comment]
> Grundsätzlich habe ich ja nichts gegen eine abweichende Lieferanschrift.

Vorher fandest du das "sinnvoll". Jetzt schreibst du: 

> "Felder, die es bereits gibt, zu entfernen ist zwar leichter als neu hinzuzufügen, aber insgesamt sollten wir es nicht übertreiben."

Was meinst du denn mit übertreiben? Ist es übertrieben, wenn der XH-Shop Kontaktangaben, die zwar nicht alle, aber doch einige Kunden angeben wollen, in separaten Feldern speichert, und diese in der bill zur **automatischen Verarbeitung** - darum gehts ja - bereitstellt?

**Comment by cmb69** [comment]
> Was meinst du denn mit übertreiben? Ist es übertrieben, wenn der XH-Shop Kontaktangaben, die zwar nicht alle, aber doch einige Kunden angeben wollen, in separaten Feldern speichert, und diese in der bill zur automatischen Verarbeitung - darum gehts ja - bereitstellt?

Die Frage ist halt wieviele solcher Felder. Nehmen wir jeden individuellen Wunsch auf, dann sind es schnell ein paar Dutzend oder gar hundert zusätzlicher Felder. Mir geht es nur darum zu verhindern, dass das **völlig** ausufert (es geht ja nicht nur um den Code, sondern auch um zusätzliche Sprachtexte, und davon gibt es schon jetzt eine ganze Menge). Daher denke ich, es wäre sinnvoll, eine möglichst große Anzahl von Anwendern anzusprechen, so dass **gemeinsam** überlegt werden kann, welche Felder hinzugefügt werden sollen, und welche nicht.

**Comment by cmb69** [comment]
> Bei meinem (Wellrad) Onlineshop wurde bei ca. 5 - 10 % der Bestellungen eine abweichende Lieferanschrift angegeben. Gründe waren dafür beispielsweise:
> 
> - Ein onlineversierter Sohn / Tochter bestellt für die betagten Offline Eltern / Großeltern"
> - Die Lieferung erfolgt an den Arbeitsplatz, da Arbeitszeit und Lieferzeiten / Postfilialen Öffnungszeiten nicht vereinbar sind
> - Das Bestellung soll als Geschenk direkt zugestellt werden.

Alles in allem gute Gründe, die die Erfassung einer abweichenden Lieferadresse rechtfertigen. Bei Punkt 3 sehe ich allerdings das Problemchen, dass man sich vielleicht noch eine Geschenkverpackung wünscht. Sollte es dafür ebenfalls eine extra Checkbox geben?

**Comment by XHalbert** [comment]
> Alles in allem gute Gründe, die die Erfassung einer abweichenden Lieferadresse rechtfertigen.

Ich würde vorschlagen, dass die "abweichende Lieferadresse" ebenfalls ein Auswahlfeld für "Land" erhält, und dass das Lieferland - wenn ausgewählt - für die Berechnung der Versandkosten herangezogen wird, anderenfalls das Land der Rechnungsadresse.

> ... noch eine Geschenkverpackung wünscht. Sollte es dafür ebenfalls eine extra Checkbox geben?

Zuerst dachte ich, das sollte ein Scherz sein. Aber nein, eine solche Checkbox wäre bestimmt sinnvoll. Die Checkbox könnte der Shopnutzer in der Sprachdatei anpassen, beispielsweise "Geschenkverpackung gewünscht", oder "eilige Bestellung" oder sonstwas. Steht nichts in der Sprachdatei, ist die Checkbox ausgeblendet.

PS: Sollen wir das Thema noch im Forum diskutieren?

**Comment by cmb69** [comment]
> Sollen wir das Thema noch im Forum diskutieren?

Ja, bitte, unbedingt. Ich fände es unglücklich, wenn wir nach und nach immer ein paar weitere Felder ergänzen, nur weil einer das wünscht, und keiner ausdrücklich was dagegen hat. Besser im Forum mal brainstormen, welche Felder überhaupt gewünscht werden (am besten in einem eigenen Thread, der gerne über Wochen oder gar Monate aktiv sein kann), und dann entscheiden, wie es gelöst wird.

Okay, siehe https://cmsimpleforum.com/viewtopic.php?f=16&t=13055.
